packages/tradershub/src/features/cfd/screens/DynamicLeverage/DynamicLeverageTableColumnHeader.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import { CaptionText, Text } from '@deriv/quill-design';
+
+type TDynamicLeverageTableColumnHeader = {
+    subtitle: string;
+    title: string;
+};
+
+export const DynamicLeverageTableColumnHeader: FC<TDynamicLeverageTableColumnHeader> = ({ subtitle, title }) => (
+    <div className='flex flex-col'>
+        <Text align='center' bold size='sm'>
```

Comment:
Do we need align="center" here?

Reply:
Why are we adding same thing in two different PRs?
This is also included in https://github.com/binary-com/deriv-app/pull/12747/files

Reply:
Created this branch off that branch, It won't show up once that branch is merged.
We do need center alignment, either we can do this with `align='center'` or with tailwind class `text-center`